recipes/cpr/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -euo pipefail
+
+mkdir -p build
+cd build
+cmake -GNinja \
```

Comment:
You can avoid most of this commands by passing the `$CMAKE_ARGS` option:
```suggestion
cmake -GNinja $CMAKE_ARGS \
```

See https://conda-forge.org/docs/maintainer/knowledge_base/#using-cmake and https://conda-forge.org/docs/user/faq/#why-dont-the-cc-compilers-automatically-know-how-to-find-libraries-installed-by-conda  .

Reply:
In the past I saw different staged-recipes maintainers that have different opinions on this. I do not really feel strongly about this, but if you really want to keep all the options for readability, I think you can have both `${CMAKE_ARGS}` and the additional options. Having `${CMAKE_ARGS}` is important to ensure that cross-compilation works fine.

Reply:
I would also say that probably the second option even if less readable probably will minimize the risk of having bikeshedding on this PR reviews. : )